### DISCUSSION TITLE
doris 重启be和fe后， be的backend id改变导致所有的搜索都找不到

### DISCUSSION THREAD

**Discussion by cst4049** [apache/doris#26178] [A - General / Q&A]
version: 2.0.2
k8s 部署 1be 1fe

重启后节点backend id 由10002 ==> 18005

查询全部显示10002backends not alive

**Comment by intelligentfu** [comment]
你是怎么部署的，通过直接下发statefulset的yaml还是使用推荐的doris-operator这种k8s外接controller模式部署的。
image的用的是dockerhub地址发一下。
